src/lsptoolshost/unitTesting.ts

```diff
@@ -6,47 +6,52 @@
 import * as vscode from 'vscode';
 import * as languageClient from 'vscode-languageclient/node';
 import { RoslynLanguageServer } from './roslynLanguageServer';
-import { RunTestsParams, RunTestsPartialResult, RunTestsRequest } from './roslynProtocol';
+import { RunTestsParams, RunTestsPartialResult, RunTestsRequest, TestProgress } from './roslynProtocol';
 
 export function registerUnitTestingCommands(
     context: vscode.ExtensionContext,
     languageServer: RoslynLanguageServer,
     dotnetTestChannel: vscode.OutputChannel
 ) {
     context.subscriptions.push(
-        vscode.commands.registerCommand('dotnet.test.run', async (request) =>
-            runTests(request, languageServer, dotnetTestChannel)
+        vscode.commands.registerCommand(
```

Comment:
general fixes in here to allow the tests to see the result of the run tests command.